package.json

```diff
@@ -55,7 +56,7 @@
         "permissionName": "ui-circulation.settings.loan-history",
         "displayName": "Settings (Circ): Can view loan history",
         "subPermissions": [
-          "settings.circulation.enabled"
+          "payments.collection.get"
         ]
```

Comment:
Add `"visible": true` below `subPermissions` like it is done for other permissions there.